Notepad/Theme.swift

```diff
@@ -64,10 +64,14 @@ public struct Theme {
 
             path = path3
         }
+        else if let path4 = bundle.path(forResource: "\(name)", ofType: "json") {
```

Comment:
since `name` is a String, you can omit the String interpolation:

```suggestion
        else if let path4 = bundle.path(forResource: name, ofType: "json") {
```